modules/experimental-layers/src/mesh-layer/mesh-layer.js

```diff
@@ -243,16 +288,68 @@ export default class MeshLayer extends Layer {
     }
   }
 
-  // yaw(z), pitch(y) and roll(x) in radians
-  calculateInstanceRotations(attribute) {
-    const {data, getYaw, getPitch, getRoll} = this.props;
-    const {value, size} = attribute;
+  calculateInstanceXform() {
+    const {data, getYaw, getPitch, getRoll, getScale, getTranslation} = this.props;
+    let matrixData = this.state.matrixData;
+
     let i = 0;
     for (const point of data) {
-      value[i++] = getRoll(point) * RADIAN_PER_DEGREE;
```

Comment:
```
let instanceModelMatrix = getMatrix(object);
if (!instanceModelMatrix) {
  ...
```

Nit: Not part of your current changes, but I would rename the iteration variable `point` => `object`